docs/01_nodeos/06_logging/index.md

```diff
@@ -81,7 +81,7 @@ The logging library built into EOSIO currently supports the following loggers:
 - `http_plugin` - detailed logging for the http plugin.
 - `producer_plugin` - detailed logging for the producer plugin.
 - `state_history` - detailed logging for state history plugin.
-- `transaction_success_tracing` - detailed log that emits successful verdicts from relay nodes on the P2P network.
+- `transaction_tracing` - detailed log that emits successful/unsuccessful verdicts from relay nodes on the P2P network.
```

Comment:
In the develop branch, the logger is rename to `transaction_success_tracing`. The `transaction_tracing` logger remained in 2.0.x branch for backward compatibility reason.